index.html

```diff
@@ -109,20 +115,20 @@ <h1 class="items--title">Slick and beautiful design</h1>
             src="./images/imac.png"
             alt="Imac screen image"
           />
-          <div class="description--text">
+          <article class="description--text">
             <h1 class="description--title">Simple and beautiful</h1>
             <p class="description--spec">
               Use Nova theme for your next web project. It is completely
               customizable so you can change any of the sections to fit your
               needs. Nova Theme is Free for any kind of use, personal and
               commercial. Have fun and good luck!
             </p>
-          </div>
+          </article>
         </div>
       </section>
       <!-- End of description section -->
       <!-- Pricing plan section -->
-      <section class="pricing">
+      <header class="pricing">
```

Comment:
Zwróć uwagę numer linii do której robię komentarz - `<header>` miał zastąpić `<div>`, a nie `section`.

PS. Przeglądaj uwagi na GH - jest wygodniej, zobacz:

![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/e1a65ff1-0af9-4710-8bdd-070aed85f97b)